EventFilter/GEMRawToDigi/python/gemUnpacker_cfi.py

```diff
@@ -3,5 +3,5 @@
 muonGEMDigis = cms.EDProducer("GEMRawToDigiModule",
     InputLabel = cms.InputTag("rawDataCollector"),
     UnpackStatusDigis = cms.bool(False),
-    useDBEMap = cms.bool(False),
+    useDBEMap = cms.bool(True),
```

Comment:
is this change in the master already?

Reply:
no. 
and this file isnt used and should be removed.